### PR TITLE
Check in configs to support RBE

### PR DESCRIPTION
**Pull request by fejta**
ref https://github.com/istio/test-infra/issues/1396

* `.bazelversion` is for use with [bazelisk](https://github.com/bazelbuild/bazelisk)
* Copied most `.bazelrc` values from https://github.com/kubernetes/test-infra/blob/master/.bazelrc
* Fixed implicit bootstrap dependency on yaml library

Note `--remote_instance_name=projects/istio-testing/instances/default_instance` does not yet exist so won't work just yet, but I have tests passing on my RBE cluster.

### CODE REVIEW

**Comment by fejta** [comment]
```console
$ ~/src/gh/istio-test-infra$ bazel clean --expunge && time bazel test //...

INFO: Elapsed time: 71.721s, Critical Path: 25.19s
INFO: 1195 processes: 1193 remote cache hit, 2 remote.
INFO: Build completed successfully, 1259 total actions
//boskos/gcp:go_default_test                                    (cached) PASSED in 0.1s
//prow/config:go_default_test                                   (cached) PASSED in 0.1s
//sisyphus:go_default_test                                      (cached) PASSED in 0.1s
//toolbox/metrics:go_default_test                               (cached) PASSED in 0.1s
//toolbox/metrics/coverage:go_default_test                      (cached) PASSED in 0.2s
//docker/istio_builders/prow_builder/bootstrap:bootstrap_test            PASSED in 11.1s

Executed 1 out of 6 tests: 6 tests pass.
There were tests whose specified size is too big. Use the --test_verbose_timeout_warnings command line option to see which ones theINFO: Build completed successfully, 1259 total actions

real	1m11.792s
user	0m0.053s
sys	0m0.009s
```

File: prow/cluster/jobs/istio/test-infra/istio.test-infra.master.yaml [hunk 1]
```diff
@@ -37,6 +37,8 @@ presubmits:
     always_run: true
     decorate: true
     path_alias: istio.io/test-infra
+    labels:
+      preset-service-account: "true"
```
Comment (fejta) [inline prow/cluster/jobs/istio/test-infra/istio.test-infra.master.yaml:41]: We cannot actually turn on `build:ci --config=remote` until we merge this

**Review by howardjohn — APPROVED** [review]
/lgtm

File: .bazelrc [hunk 2]
```diff
@@ -6,5 +6,42 @@ build --host_force_python=PY2
...
+build:remote26 --spawn_strategy=remote
+build:remote26 --strategy=Javac=remote
+build:remote26 --strategy=Closure=remote
+build:remote26 --strategy=Genrule=remote
+build:remote --config=remote26
+
 # TODO(fejta): remove this once --config=ci actually does something else
 build:ci --features=race
+# Config we want to use in ci
+#build:ci --config=remote --config=ci-instance --google_credentials=/etc/service-account/service-account.json
```
Comment (fejta) [inline .bazelrc:47]: Will uncomment this line in the next PR, at which point the presubmit will be using RBE

**Pull request merged**